style.css

```diff
@@ -0,0 +1,792 @@
+  margin: 20px;
+}
+
+.photo-2 {
+  background-image:
+    linear-gradient(
+      to right,
+      rgba(235, 64, 52, 0.7),
+      rgba(230, 55, 44, 0.7)
+    ),
+    url(images/Conferences.jpg);
+  height: 300px;
+  margin: 20px;
+}
+
+.partner {
+  background-color: #333;
+  height: 200px;
+  display: flex;
+  flex-direction: column;
+}
+
+.line1 {
+  width: 8%;
+  background-color: #bf311f;
+  height: 1px;
+  border: none;
+  margin-left: 100px;
+}
+
+.companies {
+  display: flex;
+  gap: 10px;
+  margin: 10px;
+  justify-content: space-between;
+}
+
+.mozilla {
+  color: #818181;
+  margin-top: 10px;
+  font-size: 20px;
+  font-weight: bold;
+  letter-spacing: 0.1rem;
+  text-align: left;
+}
+
+.youtube {
+  display: inline;
+  color: #818181;
+  margin-top: 10px;
+  font-size: 20px;
+  font-weight: bolder;
+  letter-spacing: 0.1rem;
+  text-align: left;
+}
+
+.google,
+.meta,
+.airbnb {
+  color: #808080;
+  font-size: 15px;
+  font-weight: 200;
+  margin-top: 10px;
+  letter-spacing: 0.1rem;
+  gap: 10px;
+}
+
+.partners {
+  margin-bottom: 10px;
+  padding-top: 20px;
+  color: #d9d9d9;
+  text-align: center;
+  font-weight: 900;
+  font-size: 30px;
+}
+
+footer {
+  height: 70px;
+}
+
+.footer {
+  text-align: center;
+  margin-top: 20px;
+  font-weight: 400;
+}
+
+@media (min-width: 768px) {
+  .header {
+    display: flex;
+    background-color: #333;
+    height: 40px;
+    justify-content: end;
+  }
+
+  .icon {
+    text-align: center;
+    font-size: 10px;
+    width: 10px;
+    height: 10px;
+  }
+
+  .fa-twitter,
+  .fa-facebook {
+    background: #333;
+    color: white;
+  }
+
+  .header ul {
+    list-style: none;
+    display: flex;
+    gap: 15px;
+    justify-content: end;
+    margin: 10px;
+    color: #fff;
+    width: 100%;
+    margin-right: 200px;
+  }
+
+  .fa .fa-twitter {
+    background: #333;
+    color: #fff;
+  }
+
+  .logonav {
+    display: block;
+    margin-left: 20px;
+    width: 150px;
+    height: 70px;
+  }
+
+  nav {
+    display: flex;
+    box-sizing: border-box;
+  }
+
+  .menu {
+    position: relative;
+    list-style: none;
+    display: flex;
+    gap: 15px;
+    justify-content: end;
+    float: right;
+    margin-left: 370px;
+    margin-top: 20px;
+  }
+
+  .ul-menu {
+    color: #333;
+    margin: 10px;
+  }
+
+  .menu li a:hover {
+    color: #bf311f;
+  }
+
+  .child-7 {
+    margin-left: 60px;
+    color: #bf311f;
+    border: #bf311f 4px solid;
+    padding: 5px;
+    margin-bottom: 10px;
+    text-decoration: none;
+  }
+
+  .bar {
+    display: none;
+  }
+
+  .wrapper {
+    width: 100%;
+  }
+
+  .intro {
+    height: 580px;
+  }
+
+  .title-home {
+    display: none;
+  }
+
+  .title-about,
+  .h1 {
+    font-family: inherit;
+    font-size: 60px;
+    font-weight: 900;
+    line-height: 3rem;
+    letter-spacing: 0.2rem;
+    padding-top: 50px;
+  }
+
+  .h1 {
+    text-align: center;
+    margin-left: 400px;
+    margin-right: 400px;
+  }
+
+  .info {
+    margin-left: 320px;
```

Comment:
- [ ] Kindly make sure to align this paragraph to the center.
_You can do this by replacing `margin-left` with `margin: 0 auto`._ 😄